bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -35,10 +34,11 @@
 using std::set;
 using std::map;
 using std::string;
+using concordMetrics::StatusHandle;
+using concordMetrics::GaugeHandle;
+using concordMetrics::CounterHandle;
```

Comment:
[Minor]: I think it may not be a good idea to put these in header files. Can we put them in the cpp file instead and use the full name in the header?

Reply:
Is it kinda good practice?

Reply:
so why not to replace all `using` declarations above?

Reply:
I think we should, especially the `std` ones. I think we may do it in a separate PR.